body.tex

```diff
@@ -46,7 +48,7 @@ \subsection{Annual Reporting}
 
 The FWP is delivered in June of the current fiscal year, and covers the federal budget request for the next two fiscal years.
 The FWP is previewed to SLAC management and DOE in February in advance of the final delivery in June.
-Because the POP for NSF and FWP for DOE are out of phase, Rubin does high level planning in early Q2 of the financial year (January and February).
+Because the POP for NSF lags the FWP for DOE by one year, and their submission dates differ by several months, Rubin does high level planning in early Q2 of the financial year (January and February) and plans 2 years ahead, as required for the SLAC FWP.
```

Comment:
```suggestion
Because the POP for NSF lags the FWP for DOE by one year, and their submission dates differ by several months, Rubin does high level planning in early Q2 of the fiscal year (January and February) and plans 2 years ahead, as required for the SLAC FWP.
```

Reply:
Another one, thanks!